spec/kit_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../lib/all'
+
+describe 'kit_spec' do
+  Dir['./lib/engines/hckinstall/kits/*.json'].each do |json_file|
+    kit_url = AutoHCK::Models::Kit.from_json_file(json_file).download_url
+    if kit_url.nil?
+      next
+    end
+
+    it "Kit for #{json_file} can be downloaded" do
+      content = String.new
+
+      catch 1 do
+        HTTPClient.get_content(kit_url) do |chunk|
+          content << chunk
+          throw 1 if content.size > 32_773
```

Comment:
Let `catch` create a tag for us instead of using 1. Using 1 is unsafe as httpclient may, in theory, internally have `catch 1`.

Reply:
I found we can just use `break` here... my bad assuming `break` doesn't work.